docs/index.bs

```diff
@@ -2557,6 +2565,12 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
         Note: Keep this definition in sync with [=fetch a classic worker-imported script=].
 
+  A <dfn id="dfn-race-result">race result</dfn> is a tuple of a [=/race response=] and [=/used route=].
+
+  A [=/race result=] has an associated <dfn export id="dfn-race-response">race response</dfn> (a [=/response=]).
+
+  A [=/race result=] has an associated <dfn export id="dfn-used-route">used route</dfn> (a [=RouterSourceEnum=]).
```

Comment:
maybe `{{RouterSourceEnum}}` instead because this is the word showed up in the IDL.

Reply:
Considering the consistency, it might be good to put "used route" under the "race result".  I mean `<dfn export for="race result">used route</dfn>`